index.html

```diff
@@ -0,0 +1,1080 @@
+                            <div class="collapse navbar-collapse d-none d-sm-none d-md-none d-lg-flex text-nowrap" style="margin-left: 10px;">
+                                <ul class="nav navbar-nav d-flex d-md-none d-lg-flex d-xl-flex">
+                                    <li class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>U.S.</strong></a></li>
+                                    <li
+                                        class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>World</strong></a></li>
+                                        <li
+                                            class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Business</strong></a></li>
+                                            <li
+                                                class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Tech &amp; Science</strong></a></li>
+                                                <li
+                                                    class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Culture</strong></a></li>
+                                                    <li
+                                                        class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Newsgeek</strong></a></li>
+                                                        <li
+                                                            class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Sports</strong></a></li>
+                                                            <li
+                                                                class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Health</strong></a></li>
+                                                                <li
+                                                                    class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>The Debate</strong></a></li>
+                                                                    <li
+                                                                        class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Vantage</strong></a></li>
+                                                                        <li
+                                                                            class=" nav-header-weather nav-item"><a class="nav-link" href="#" ><strong>Weather</strong></a></li>
+                                                                            <li class="nav-item d-flex"><a class="nav-link" href="#">Search</a><i class="fa fa-search"
+                                                                                    style="color: rgb(99,99,99); margin-top: 10px;"></i></li>
+                                </ul>
+                        </div>
```

Comment:
- [ ] Try to maintain the correct indentation through the whole file. [OPTIONAL]